tests/pytests/test_issues.py

```diff
@@ -560,4 +560,37 @@ def test_mod_4207(env):
   conn.execute_command('HSET', 'address:4', 'country', 'Israel')
 
   env.expect('FT.SEARCH', 'idx1', '*', 'NOCONTENT').equal([3, 'address:1', 'address:2', 'address:4'])
-  env.expect('FT.SEARCH', 'idx2', '*', 'NOCONTENT').equal([3, 'address:1', 'address:2', 'address:3'])
\ No newline at end of file
+  env.expect('FT.SEARCH', 'idx2', '*', 'NOCONTENT').equal([3, 'address:1', 'address:2', 'address:3'])
+
+def test_mod_4232(env):
+  conn = getConnectionByEnv(env)
+
+  env.cmd('FT.CREATE', 'idx1', 'SCHEMA', 'business', 'TEXT')
+  env.cmd('FT.CREATE', 'idx2', 'SCHEMA', 'country', 'TEXT')
+  conn.execute_command('HSET', 'addr:1', 'business', 'foo', 'country', 'USA')
+
+  env.cmd('FT.SEARCH', 'idx1', '*', 'NOCONTENT', 'DIALECT', 3)
```

Comment:
We should also test `FT.AGGREGATE` and `FT.SPELLCHECK` (reuse test code in a function)

About `FT.EXPLAIN` and `FT.EXPLAINCLI`, also accepting a `DIALECT` arg,
it should be clear how they are affecting the dialect stats.
Seems to me these 2 should not be counted (not real queries)

@K-Jo Agree?

Reply:
Ok with not counting these